fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -293,15 +457,82 @@ if [ ! -e "$fMRIFolder" ] ; then
   log_Msg "mkdir ${fMRIFolder}"
   mkdir "$fMRIFolder"
 fi
-cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
+${FSLDIR}/bin/imcp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName"
+
+# --- Do slice time correction if indicated
+if [ $DoSliceTimeCorrection = "TRUE" ] ; then
+    log_Msg "Running slice timing correction using FSL's 'slicetimer' tool"
+    TR=`${FSLDIR}/bin/fslval "$fMRIFolder"/"$OrigTCSName" pixdim4`
```

Comment:
Would suggest modifying the log_Msg's as well:
```
log_Msg "Running slice timing correction using FSL's 'slicetimer' tool ..."
log_Msg "... $fMRIFolder/$OrigTCSName will be a slice-time-corrected version of the original data"
```

Reply:
Done.